linux-creator.sh

```diff
@@ -0,0 +1,11 @@
+# cleaning Linux's build folder
```

Comment:
Is there a better place to store these script files than the root directory? How about in `installers/linux`?

Also, shell script files should start with a `#!/bin/sh` or `#!/bin/bash` line.

Reply:
moved to scripts folder.